src/types/mock.ts

```diff
@@ -2,29 +2,29 @@ import { RequestMethod } from ".";
 
 // TODO: Convert into classes later on
 export interface Mock extends MockMetadata {
-    responses: Response[] // Right now we are keeping only 1 response
+  responses: Response[]; // Right now we are keeping only 1 response
 }
 
 // Useful when fetching list of mocks. Saves network bandwith
 export interface MockMetadata {
-    id : string
-    name?: string
-    desc ?: string
-    method: RequestMethod
-    endpoint: string
-    ownerId?: string
```

Comment:
I think it will be fine to keep these optional. This makes it possible to use this package to create dynamic, unnamed mocks on the fly by non-logged in users. 

We should keep the package extensible that way. 

If the app requires the `ownerId` be required, we can enforce that on the consumer in some way

Reply:
Then in that case we should keep this as null ie `ownerId: string | null`.

Reply:
Okay, keeping it optional then